libbeat/metric/system/diskio/diskstat_linux_test.go

```diff
@@ -34,6 +34,23 @@ func Test_GetCLKTCK(t *testing.T) {
 	assert.Equal(t, uint32(100), GetCLKTCK())
 }
 
+func Test32BitRollover(t *testing.T) {
+	var maxUint32 uint64 = 4_294_967_295
```

Comment:
Same thing here, you could use the constant `math.MaxUint32`.
```suggestion
	var maxUint32 uint64 = math.MaxUint32
```